includes/rrdtool.inc.php

```diff
@@ -166,7 +166,11 @@ function rrdtool_graph($graph_file, $options) {
 function rrdtool($command, $filename, $options) {
     global $config, $debug, $rrd_pipes, $console_color;
 
-    if ($config['rrdcached'] && ($config['rrdtool_version'] >= 1.5 || $command != "create")) {
+    if ($config['rrdcached'] &&
+        ($config['rrdtool_version'] >= 1.5.5 ||
```

Comment:
Does that construct even work?  Is it a string comparison or numeric?  Or does PHP have some special magic for version numbers?